packages/terra-embedded-content-consumer/tests/jest/__snapshots__/EmbeddedContentConsumer.test.jsx.snap

```diff
@@ -1,5 +1,23 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`sets appropriate config option when scrolling is false 1`] = `
```

Comment:
Should we see the frame config options here in the snap file or are these tests just to verify that the prop is set appropriately?

Reply:
Yeah, it doesn't render the frames since that is several levels down. I've updated it to test the props are set correctly instead with `shallow`.